backend/apps/auth/tests/conftest.py

```
@@ -0,0 +1,96 @@
+os.environ["GOOGLE_CLIENT_SECRET"] = "test-google-secret"
+
+# 모델 import
+from src.models.database import Base, get_db
+from src.models.user import User, OAuthAccount, Image
+from src.server import app
+
+# SessionMiddleware 추가
+app.add_middleware(SessionMiddleware, secret_key="test-session-secret")
+
+# ⭐ 임시 파일로 SQLite DB 생성
+TEST_DB_FILE = tempfile.mktemp(suffix=".db")
```

Ruff (0.14.0)

Use of insecure and deprecated function (`mktemp`)

(S306)